Documentation/openshiftdocs/modules/nodeexportercollectorcpufreqconfig.adoc

```diff
@@ -0,0 +1,25 @@
+// DO NOT EDIT THE CONTENT IN THIS FILE. It is automatically generated from the 
+	// source code for the Cluster Monitoring Operator. Any changes made to this 
+	// file will be overwritten when the content is re-generated. If you wish to 
+	// make edits, read the docgen utility instructions in the source code for the 
+	// CMO.
+	:_content-type: ASSEMBLY
+
+== NodeExporterCollectorCpufreqConfig
+
+=== Description
+
+The `NodeExporterCollectorCpufreqConfig` resource works as a on/off switch for the cpufreq collector of the Node Exporter.
```

Comment:
```suggestion
The `NodeExporterCollectorCpufreqConfig` resource works as a on/off switch for the `cpufreq` collector of the `node-exporter` agent.
```

Reply:
Node exporter works more as a daemon than an agent, so it is called a daemon in the new commit.

Reply:
I guess that @bburt-rh suggested "node-exporter agent" because this is the current terminology in the [OCP docs](https://docs.openshift.com/container-platform/4.11/monitoring/monitoring-overview.html).

Reply:
thanks, the new term "agent" is used in the new commit just pushed 👍

Reply:
@raptorsun - @simonpasquier is correct. I'm trying to stay consistent with the OCP docs. If "daemon" is more accurate than "agent", then I should probably write up bug to change it in the core docs.

Reply:
Thanks for the clarification, @bburt-rh 
Let's keep it "agent". I would like to avoid this simple feature PR compromising the consistency of our documentation. We can always change it afterwards if that causes confusion for our users :) 
The reason I feel a "daemon" is a better fit is that the "agent" is often mentioned in [MacOS as a daemon per user session](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html).